src/main/content/_assets/css/doc-header.css

```diff
@@ -1,3 +1,209 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+ .navbar-default .navbar-nav > li > a {
     margin-left: 10px;
+}
+
+#header_background{
+    padding-top: 121px;
+    background-color: #313653;
+    background-image: none;
+}
+
+#doc_logo,
+#doc_title {
+    float: left;
+}
+
+#doc_title {
+    padding-left: 15px;
+    font-family: BunueloLight;
+    font-size:30px;
+    color:#f1f4fe;
+    letter-spacing:0;
+    text-align:left;
+}
+
+.reference_dropdown {
+    font-size:14px;
+    color:#c8d2d2;
+    letter-spacing:0;
+    text-align:center;
+    border: none;
+    background: inherit;
+}
+
+/* Override the dropdown button colors */
+#navbar .btn-default.active, #navbar .btn-default:active, #navbar .open > .dropdown-toggle.btn-default{
+    color: white;
+    border: none;
+    background-color: inherit;
+    box-shadow: none;
+    -webkit-box-shadow: none;
+}
+
+#reference_dropdown_container {
+    margin-top: 9px;
+    display: inline-block;
+}
+
+.reference_dropdown:hover, .reference_dropdown:focus, .reference_dropdown:active {
+    background-color: inherit;
+    border: none;
+    color: white;
+}
+
+.dropdown-menu {
+    background: #F1F4FE;
+    box-shadow: 0 2px 4px 0 rgba(0,0,0,0.25);
+    border-radius: 5px;
+}
+
+#reference_dropdown_container .dropdown-menu > li {
+    font-size: 15px;
+    color: #24243B;
+    letter-spacing: 0;
+    background-color: inherit;
+}
+
+#reference_dropdown_container .dropdown-menu > li > a:hover, #reference_dropdown_container .dropdown-menu > li > a:focus {
+    color: #5D6A8E;
+    background-color: inherit;
+}
+
+#breadcrumb_row {
+    height: 41px;
+}
+
+#breadcrumb_row .breadcrumb {
+    background-color: #24243A;
+    padding-left: 30px;
+    height: 100%;
+    margin-bottom: 0;
+    border-radius: 0;
+}
+
+#breadcrumb_row a {
+    color:#c8d6fb;
+}
+
+/* Unclickable link */
+#breadcrumb_row a.inactive_link {
+    color:#FFFFFF;
+}
+
+#breadcrumb_row a:hover, #breadcrumb_row a:focus,
+#breadcrumb_row > .breadcrumb > li.active, #breadcrumb_row > .breadcrumb > li.active + li:before {
+    color:white;
+}
+
+/* Change default bootstrap breadcrumb separator */
+.breadcrumb > li + li:before {
+    content: "> ";
+}
+
+.breadcrumb > li + .javadoc_version::before{
+    content: none;
+}
+
+#breadcrumb_line{
+    margin: 0px 10px;
+    font-size: 20px;
+    color: white;
+    letter-spacing: 0;
+    line-height: 26px;
+    vertical-align: middle;
+}
+
+#breadcrumb_version{
+    font-family:Asap;
```

Comment:
Do we need to specify `Asap`?  Shouldn't `Asap` be the default font if no `font-family` is set?